scripts/test/Muon/utilities/muon_workspace_wrapper_directory_test.py

```diff
@@ -65,83 +65,14 @@ def assert_workspace_in_group(self, workspace_name, group_name):
         group = simpleapi.mtd.retrieve(group_name)
         self.assertIn(workspace_name, group.getNames())
 
-    # ----------------------------------------------------------------------------------------------
-    # Test add_directory_structure function
-    # ----------------------------------------------------------------------------------------------
-
-    def test_that_passing_empty_list_has_no_effect(self):
```

Comment:
do we need these or similar tests?

Reply:
These tests are to do with creating nested directories of workspace groups. This is functionality that has been removed as doesn't work with project save or workspace history.